### ISSUE TITLE
Slider or SpinBox working with alpha channel

### ISSUE THREAD

**Issue by smolnp** [maifeeulasad/Paint#20]
To be able to change the transparency of the selected line color or fill.

**Comment by maifeeulasad** [comment]
@smolnp check it out, will this work now ?

**Comment by smolnp** [comment]
Yes, that's what I meant. But the right-click responsible for the background inside the shape does not work for you. Also the alpha parameter needs to be added to "Primary color" and "Secondary color". Or add two SpinBoxes (one changes the transparency of the line with the left mouse, the second changes the transparency of the line with the right mouse and the background inside the rectangle) with changing the transparency of the directly selected color when working with a picture, then it will not be needed in the color selection tools.

**Comment by maifeeulasad** [comment]
@smolnp i couldn't get the left-right mouse selection part.

**Comment by smolnp** [comment]
Here you can clearly see which tool for changing the transparency I'm talking about: https://github.com/Alpha303/qPaint
